internal/cli/default_setter_opts.go

```diff
@@ -36,7 +36,7 @@ import (
 type ProjectOrgsLister interface {
 	Project(id string) (interface{}, error)
 	Projects(*atlas.ListOptions) (interface{}, error)
-	Organization(id string) (interface{}, error)
+	Organization(id string) (*atlas.Organization, error)
```

Comment:
[q] Did we consider renaming the current `Organization` interface to `OrganizationDeprecated` and creating a new `Organization` interface that uses only the SDK v2?